cmd/system-probe/probe.go

```diff
@@ -161,14 +161,20 @@ func getClientID(req *http.Request) string {
 }
 
 func writeConnections(w http.ResponseWriter, cs *ebpf.Connections) {
-	buf, err := easyjson.Marshal(cs)
-	if err != nil {
+	jw := &jwriter.Writer{}
```

Comment:
The writer has its buffer structure which could be reused (`DumpTo` resets it)  if we can do it in a thread-safe way.

Reply:
I was originally going to do this, but after I looked at the code of  `DumpTo` I saw that all the allocated memory that stuck around lived inside the global pool, not the writer. So sharing the writer would not cut down on too many allocations.